EapWebInterface/src/main/java/de/hpi/unicorn/application/pages/input/generator/GeneratePanel.java

```diff
@@ -242,4 +307,124 @@ private void addSubmitButton() {
         final Button submitButton = new Button("submitButton");
         layoutForm.add(submitButton);
     }
+
+    /**
+     * Add a button to export the input values given via input fields
+     */
+    private void addExportValuesButton() {
+        final AjaxLink exportButton = new AjaxLink<Void>("exportValuesButton") {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Reduce this anonymous class number of lines from 29 to at most 20, or make it a named class. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://bpt-lab.org/sonarqube/coding_rules#rule_key=squid%3AS1188)